scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/ForkEnvironmentScriptExecutor.java

```diff
@@ -63,6 +63,8 @@ public ScriptResult executeForkEnvironmentScript(TaskContext context, PrintStrea
         VariablesMap variables = new VariablesMap();
         variables.setInheritedMap(taskContextVariableExtractor.getAllNonTaskVariables(context));
         variables.setScopeMap(taskContextVariableExtractor.getScopeVariables(context));
+        Map<String, Serializable> JobMap = taskContextVariableExtractor.getJobMapVariables(context);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)